packages/plugins/plugin-thread/src/components/ThreadComplementary.tsx

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright 2024 DXOS.org
+//
+import React, { useEffect, useMemo } from 'react';
+
+import { LayoutAction, type Plugin, useIntentDispatcher, useResolvePlugins } from '@dxos/app-framework';
+import { type ThreadType } from '@dxos/plugin-space';
+import { fullyQualifiedId } from '@dxos/react-client/echo';
+import { ScrollArea } from '@dxos/react-ui';
+import { createAttendableAttributes, useAttentionContext } from '@dxos/react-ui-attention';
+import { nonNullable } from '@dxos/util';
+
+import { CommentsContainer, CommentsHeading } from './';
```

Comment:
Fix lint to prevent `./` import (and pls fix this line)

Reply:
You would prefer '../components'?